corehq/apps/accounting/tasks.py

```diff
@@ -220,6 +220,18 @@ def update_subscriptions():
     warn_active_subscriptions_per_domain_not_one()
     warn_subscriptions_without_domain()
 
+    check_credit_line_balances.delay()
+
+
+@task
+def check_credit_line_balances():
+    for credit_line in CreditLine.objects.all():
```

Comment:
How many credit lines are there on prod? If there are a lot, should this get filtered more?

May  also want to prefetch (or select_related, I can't remember) whatever `creditadjustment_set` is if there are a lot of these

Reply:
This runs pretty quickly (tested it in a private release) so for now I'm not worried about performance.